hack/generate-latest-csv.sh

```diff
@@ -16,7 +16,7 @@ export CSV_VERSION=0.0.1
 export ROOK_IMAGE=${ROOK_IMAGE:-"rook/ceph:v1.1.1"}
 export NOOBAA_IMAGE=${NOOBAA_IMAGE:-"noobaa/noobaa-operator:1.1.1"}
```

Comment:
should we update the noobaa operator image used for the alpha 0.0.1 ocs-operator CSV here?

Reply:
updated and generated csv. thanks